src/components/helpers.js

```diff
@@ -28,3 +28,8 @@ export default function webexComponentClasses(
     ...otherClasses,
   });
 }
+
+const isFirefox = () => navigator.userAgent.indexOf('Firefox') !== -1; // detect Firefox browser
+const isSafari = () => navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1; // detect Safari browser
+
+export const isSpeakerSupported = !(isFirefox() || isSafari());
```

Comment:
I think a safer (and future-proof, eg when setSinkId will be available in this browsers, probably soon in firefox) would be to check for the existence of setSinkId on a media element.

``js
!!document.createElement('audio').setSinkId;